calendar-bundle/src/Resources/contao/classes/Events.php

```diff
@@ -333,7 +333,6 @@ protected function addEvent($objEvents, $intStart, $intEnd, $intBegin, $intLimit
 		// Display the "read more" button for external/article links
 		if ($objEvents->source != 'default')
 		{
-			$arrEvent['details'] = true;
 			$arrEvent['hasDetails'] = true;
```

Comment:
Since `hasDetails` is still true, this would still not show the teaser in the default `event_list` template for example.

Reply:
It is not supposed to show the teaser. There is an `event_teaser` template if you want to accomplish that.

Reply:
But `hasDetails` needs to be true for other event templates. I think the `event_list` template has to be adjusted to somethink like `<?php if ($this->hasDetails && !empty($this->details)): ?>`

Reply:
Fixed in ef11121837232a7cfba628d60e0a6b27836c5b0c.